device/mellanox/x86_64-nvidia_sn5400-r0/platform_components.json

```diff
@@ -0,0 +1,15 @@
+{
+    "chassis": {
+        "SN5400": {
+            "component": {
+                "ONIE": { },
+                "SSD": { },
+                "BIOS": { },
+                "CPLD1": { },
+                "CPLD2": { },
+                "CPLD3": { },
+                "CPLD4": { }
```

Comment:
Why on this file there is up to CPLD4 and on platform.jason uo to CPLD6 appear ?